index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+        integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <a href="" id="logo">My Logo</a>
+            <div id="menu">
+                <i class="fa-solid fa-bars"></i>
+            </div>
+            <ul id="links">
+                <li class="portfolio-link"><a href="#">Portfolio</a></li>
+                <li class="about-link"><a href="#">About</a></li>
+                <li class="contact-link"><a href="#">Contact</a></li>
+                <li class="email-link"><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="headline">
+        <div>
+            <h1 id="headline-title">Hey There. <br>
+                I'm Mahamadou Nouridine</h1>
+            <h3 id="headline-title2">I'm A Software Developer</h3>
+            <p id="headline-description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+            <div id="headline-social">
+                <i class="fa-brands fa-github"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-brands fa-instagram"></i>
+            </div>
```

Comment:
- [ ] Kindly make sure, the list of icons uses a `ul` tag as container to group them, and `li` per icon; each icon has a link (a tag). Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using `href` attribute of the the `a` tag. example:
```html
<ul>
  <li>
     <a href="..."><i class="fa-brands fa-twitter"></i></a>
  </li>
  <li>
     <a href="..."><i class="fa-brands fa-twitter"></i></a>
  </li>
  ...
</ul>
```